depot/steps/download_step.go

```diff
@@ -91,8 +91,8 @@ func (step *downloadStep) perform() error {
 			errString = "Downloading failed"
 		}
 
-		step.emitError(fmt.Sprintf("%s\n", errString))
-		return NewEmittableError(err, errString)
+		step.emitError("%s\n", errString)
+		return NewEmittableError(err, "%s", errString)
```

Comment:
where did the NewEmittableError() function signature change? Was it just doing a fmt.sprintf() of the args passed in? 

would it make more sense to modify it directly rather than pass "%s" in every time its called?

Reply:
The signature didn't change, but golang no longer allows fmt.Sprintf(variable) with no verb-containing string, which the constructor was essentially hiding.